services/common/src/components/permits/PermitConditionForm.tsx

```diff
@@ -131,24 +130,15 @@ const PermitConditionForm: FC<PermitConditionFormProps> = ({
         });
     };
 
-    const addNewReport = async (values) => {
```

Comment:
put all the save logic within the form itself, it was kind of awkward how it was half there. Also I replaced a bunch of props (all over the place) with permit conditions context.